File: 3R/11076_GCD.cpp
#include <iostream>

using namespace std;

int GCD(int x, int y){
    if(x%y==0) return y;
    else return GCD(y, x%y);
}

int main(){
    int n;
	while(cin >> n && n!=0){
        long long G = 0;
        for(int i=1; i<n; i++){
            for(int j=i+1; j<=n; j++){
                G += GCD(i, j);
            }
        }
        cout << G << endl;
    }
}

File: 3R/11076_GCD.java
import java.util.*;

class Main{

    static int GCD(int x, int y){
        if(x % y == 0) return y;
        else return GCD(y, x%y);
    }

	public static void main(String[] args){
		Scanner sc = new Scanner(System.in);
        while(sc.hasNext()){
            int n = sc.nextInt();
            int G = 0;
            for(int i=1; i<n; i++){
                for(int j=i+1; j<=n; j++){
                    G += GCD(i, j);
                }
            }
            System.out.println(G);
        }
	}
}